keel-core/src/main/kotlin/com/netflix/spinnaker/keel/api/Resource.kt

```diff
@@ -30,9 +30,9 @@ data class Resource<T : Any>(
 
 /**
  * External representation of a resource that would be submitted to the API
- * It doesn't need to contain metadata
  */
 data class SubmittedResource<T : Any>(
+  val metadata: SubmittedMetadata,
```

Comment:
Should this be required of all resources? Should we allow no service account? I'm inclined to say we should require service account since we will always be doing things on behalf of a user.

Reply:
I think so, at least for now.